tsdf_server:
  ros__parameters:
    input:
      point_cloud: "unilidar/cloud"
    output:
      tsdf_layer: "tsdf_layer"
      mesh: "mesh"


esdf_server:
  ros__parameters:
    input:
      point_cloud: "unilidar/cloud"
    output:
      tsdf_layer: "tsdf_layer"
      esdf_layer: "esdf_layer"
      mesh: "mesh"